docs/sources/administration/configuration.md

```diff
@@ -1848,3 +1848,47 @@ default_baselayer_config = `{
 ### enable_custom_baselayers
 
 Set this to `true` to disable loading other custom base maps and hide them in the Grafana UI. Default is `false`.
+
+## [intentapi]
+
+Settings related to the Intent API.
+
+## [intentapi.server]
+
+Settings related to the Intent API HTTP server.
+
+### listen_address
+
+Which address the Intent API will be listening on, must be in `host:port` format. Default value is `127.0.0.1:8443`.
+
+### cert_file_path
+
+Path to TLS certificate that would be used by the Intent API. Since k8s effectively enforces TLS, this is required. Unset by default, make sure to set it if using the Intent API.
```

Comment:
```suggestion
Path to TLS certificate that would be used by the Intent API. Since k8s effectively enforces TLS, this is required. Unset by default, it must be set to enable Intent API.
```